### ISSUE TITLE
Example error

### ISSUE THREAD

**Issue by Borg93** [ash2shukla/streamlit-bokeh-events#7]
Hi I'm receving this error:

2021-09-02 16:00:51.015 ComponentRequestHandler: GET streamlit_bokeh_events.streamlit_bokeh_events/bootstrap.min.css.map read error
Traceback (most recent call last):

FileNotFoundError: [Errno 2] No such file or directory: 'c:\\users\\borg7803\\anaconda3\\envs\\streamlit\\lib\\site-packages\\streamlit_bokeh_events\\frontend\\build\\bootstrap.min.css.map'

**Comment by ash2shukla** [comment]
Can you give a bit more context about the error ? When are u getting the error ? A minimal reproducible example would be great.

**Comment by Borg93** [comment]
I was keen that the example you have in your README required only a pip install streamlit-bokeh-events to get that example working. However I got the error above. Am I missing somehting?

**Comment by dktcl** [comment]
Hi - I've been testing out the exmaples that @ash2shukla kindly prepared.
With the example called `widget_with_plot.py`, I receive the following error...

```
2021-11-14 15:51:15.222 ComponentRequestHandler: GET streamlit_bokeh_events.streamlit_bokeh_events/bootstrap.min.css.map read error
Traceback (most recent call last):
  File "/path/to/my/app/.venv/lib/python3.8/site-packages/streamlit/components/v1/components.py", line 321, in get
    with open(abspath, "rb") as file:
FileNotFoundError: [Errno 2] No such file or directory: '/path/to/my/app/.venv/lib/python3.8/site-packages/streamlit_bokeh_events/frontend/build/bootstrap.min.css.map'
```

Also, any charts or maps in the examples no not render, with warnings and errors in Chrome console..
![image](https://user-images.githubusercontent.com/36790933/141688596-6aadc9d6-6c02-43c4-a5bc-87db040dd59b.png)
![image](https://user-images.githubusercontent.com/36790933/141688607-f640d2b9-4ad9-4af4-bb23-1d75c475e471.png)

I'm using a Virtual Environment on Mac for python 3.8.8. with the following installed via Pip:
- streamlit              1.2.0
- streamlit-bokeh-events 0.1.2
- bokeh                  2.4.1

Happily to try and troubleshoot.

**Comment by ash2shukla** [comment]
removed sourceMap ref, this and some other minor fixes available in 0.1.3+
